cmake/test.cmake

```diff
@@ -129,6 +130,7 @@ finally:
         MAYA_PLUG_IN_PATH
         MAYA_SCRIPT_PATH
         PXR_PLUGINPATH_NAME
+        LD_LIBRARY_PATH
```

Comment:
Changes from #464 have leaked into this PR. Please remove.

Reply:
I can't.

Reply:
I can't.

Reply:
@robthebloke I looked at all the cmake changes in this PR and they make sense to me but as it was mentioned before, there was an agreement around handling LD_LIBRARY_PATH inside AL's code base rather than test.cmake

Please see Eoin's comment: 
https://github.com/Autodesk/maya-usd/pull/464#issuecomment-620939015